packages/graphql/tests/integration/deprecations/generic-filtering/types/types-localdatetime-filter-deprecated.int.test.ts

```diff
@@ -84,7 +84,7 @@ describe("LocalDateTime - deprecated filters", () => {
         "should filter based on localDT comparison, for filter %s",
         async (filter) => {
             const futureId = generate({ readable: false });
-            const future = "2025-02-18T18:10:55.462Z".split("Z")[0];
+            const future = "6025-02-18T18:10:55.462Z".split("Z")[0];
```

Comment:
... well, this was hilarious

Setting a strict date for future is a bit of a bad practice, but I guess we don't really need to worry too much about y6k here

Reply:
Absolutely, this is a cherry-picked commit from the CI fix that was failing, I didn't refactor the test tho.